Nos/Models/ReportCategory.swift

```diff
@@ -50,19 +51,19 @@ enum ReportCategoryType {
         code: "CL",
         nip56Code: .profanity
     )
-    
+
```

Comment:
These changes are kind of annoying. Is this a setting from VSCode you could change?